### PR TITLE
marsRover2 Adds a static HTML page on '/' and nodemon

### PR DESCRIPTION
**Pull request by HeatherJLewis**
**Description**
Blank home page on '/'

**Changes**
Adds a static html page on the '/' route
Adds nodemon to track changes while the server is running
Adds semicolons to index.js

### CODE REVIEW

File: frontend/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>marsRover</title>
```
Comment (Kenneth-Siu) [inline frontend/index.html:4]: What's the <title> tag used for?

File: package.json [hunk 2]
```diff
@@ -18,6 +19,9 @@
     "url": "https://github.com/HeatherJLewis/marsRover/issues"
   },
   "homepage": "https://github.com/HeatherJLewis/marsRover#readme",
+  "devDependencies": {
+    "nodemon": "^2.0.6"
```
Comment (Kenneth-Siu) [inline package.json:23]: Woooo 🎉

File: frontend/index.html (hunk 1 above)
Comment (HeatherJLewis) [reply frontend/index.html:4]: To add a label to the tab for this page in the browser.

Comment (HeatherJLewis) [reply frontend/index.html:4]: image

Comment (Kenneth-Siu) [reply frontend/index.html:4]: Yesss. In that case, since you've _chosen_ to format 'mars rover' in that way, that looks fine :)

**Review by Kenneth-Siu — APPROVED** [review]

**Pull request merged**